src/auth.test.ts

```diff
@@ -0,0 +1,219 @@
+import axios from "axios";
+
+jest.mock("axios");
+
+const mockedAxios = axios as jest.Mocked<typeof axios>;
```

Comment:
I don't think you need this here. Was just checking the mock docs - https://jestjs.io/docs/mock-functions. They actually show axios mocking as an example:
```
import axios from 'axios';
import Users from './users';

jest.mock('axios');

test('should fetch users', () => {
  const users = [{name: 'Bob'}];
  const resp = {data: users};
  axios.get.mockResolvedValue(resp);

  // or you could use the following depending on your use case:
  // axios.get.mockImplementation(() => Promise.resolve(resp))

  return Users.all().then(data => expect(data).toEqual(users));
});
```
You can just use `axios` and it'll actually be the mocked object.

Reply:
@nbragdon I get this error when using axios directly with out the `mockedAxios`. Found that as a solution researching the error.

```
    src/auth.test.ts:125:16 - error TS2339: Property 'mockResolvedValueOnce' does not exist on type '<T = any, R = AxiosResponse<T>>(url: string, data?: any, config?: AxiosRequestConfig | undefined) => Promise<R>'.

    125     axios.post.mockResolvedValueOnce(mockResponse);
```

Reply:
Making me wonder if we really need typescript on our tests...